src/renderer.cpp

```diff
@@ -38,15 +38,25 @@ namespace big
 		ImGui_ImplDX11_Init(m_d3d_device.Get(), m_d3d_device_context.Get());
 		ImGui_ImplWin32_Init(g_pointers->m_hwnd);
 
-		file font_file_path("C:/Windows/Fonts/msyh.ttc");
-		if (!font_file_path.exists())
-			font_file_path = { "C:/Windows/Fonts/msyh.ttf" };
-		auto font_file = std::ifstream(font_file_path.get_path(), std::ios::binary | std::ios::ate);
-		const auto font_data_size = static_cast<int>(font_file.tellg());
-		const auto font_data = std::make_unique<std::uint8_t[]>(font_data_size);
```

Comment:
Is assuming the drive letter is C correct here?

Reply:
I hope people have their operating system on `C:`, I think there are a lot of programs that would break if people didn't install their stuff on `C:`

Reply:
C is the default, but you can change it without any issue even though that's quite uncommon

Reply:
We can probably switch to std::getenv or smth

Reply:
Seems we can use `SYSTEMROOT` to get for example `C:/Windows/`